### PR TITLE
[Easi 4354]/ch no missing field unit test

### PR DESCRIPTION
**Pull request by StevenWadeOddball**
# EASI-4354

## Changes and Description

- Add a new unit test utility function `assertTranslationStructCoverage` to iterate through all fields on a struct, and assert that a translation exists for all fields except excluded
- Call the new unit test for each translation

## How to test this change
1. Look through all the changes
2. Verify the newly added translations
3. Verify the fields being excluded make sense to be excluded

If you want to test the behavior that will happen for a new field being added, add a new field to a struct (like model plan). Run the test and assert that is fails do to their being a new field with the db tag, but no translation.

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: mappings/translation/plan_general_characteristics.json [hunk 1]
```diff
@@ -24,6 +24,24 @@
...
+  },
+  "currentModelPlanID": {
+    "gqlField": "currentModelPlanID",
+    "goField": "CurrentModelPlanID",
+    "dbField": "current_model_plan_id",
+    "label": "What model is this a track of?",
+    "dataType": "UUID",
+    "formType": "TEXT",
+    "tableReference": "model_plan"
+  },
```
Comment (patrickseguraoddball) [inline mappings/translation/plan_general_characteristics.json:44]: I think the merge overwrote the property "hideFromReadonly"

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]

File: mappings/translation/plan_general_characteristics.json (hunk 1 above)
Comment (StevenWadeOddball) [reply mappings/translation/plan_general_characteristics.json:44]: Thanks for catching that Patrick! I'll update it

Comment (patrickseguraoddball) [reply mappings/translation/plan_general_characteristics.json:44]: It was there, was looking in the export, where it's not needed, my bad!

Comment (StevenWadeOddball) [reply mappings/translation/plan_general_characteristics.json:44]: No problem at all :)

**Review by patrickseguraoddball — APPROVED** [review]
LG thanks!

**Pull request merged**